tests/test_main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+from app.main import is_isogram
+
+
+def test_empty_string_and_result_type_bool():
+    assert is_isogram("") == True
+
+
+def test_for_upper_case_letters_and_isograms():
+    assert is_isogram("Apex") == True
+    assert is_isogram("FulL") == False
+    assert is_isogram("bAnaNA") == False
+    assert is_isogram("dermatoglyphics") == True
```

Comment:
Make 3 different tests `test_no_repeated_letters`, `test_repeated_letters`, `test_repeated_insensitive_letters` out of this test.